### DISCUSSION TITLE
User has to sign in again and again when launching app on iOS

### DISCUSSION THREAD

**Discussion by kaaloo** [ionic-team/capacitor#6326] [General]
Any insights as to why some users need to sign in again and again each time the use our app on iOS?  Our web app is hosted on Vercel and is a Next.js app. We are using supabase auth.  The only thing I can think if is that somehow the Supabase auth cookies are getting cleared but I don't see why this would affect some users and not others.  I can't reproduce the behavior on a stock iPhone. I've asked on the Supabase discord but I'm not sure there are that many developers using the same setup.

**Comment by maiieul** [comment]
Quote from @capacitor-firebase/authentication:

> What is the difference between the native and web authentication?
For web authentication, the Firebase JS SDK is used. This only works to a limited extent on Android and iOS in the WebViews (see [here](https://developers.googleblog.com/2016/08/modernizing-oauth-interactions-in-native-apps.html)). For native authentication, the native SDKs from Firebase, Google, etc. are used. These offer all the functionalities that the Firebase JS SDK also offers on the web. However, after a login with the native SDK, the user is only logged in on the native layer of the app. If the user should also be logged in on the web layer (for example to access Cloud Firestore via Firebase JS SDK), additional steps are required (see [here](https://github.com/capawesome-team/capacitor-firebase/blob/main/packages/authentication/docs/firebase-js-sdk.md)).

I don't know for sure, but I guess the webview's cookies can't be stored in the phone's cache. Would love to have confirmation on this. 

Perhaps the official cookies plugin could help?

**Comment by robmarshall** [comment]
I am also having this issue, and struggling with it. Any insight would be super helpful!

**Comment by JimmyLv** [comment]
me too
